packages/ui-toolkit/src/components/atoms/ToggleSwitch/toggleSwitch.css

```diff
@@ -18,10 +18,16 @@
   align-items: center;
   justify-content: space-between;
   cursor: pointer;
+  height: 24px;
+  width: 40px;
   position: relative;
-  transition: background-color 0.3s;
+  transition: background-color 0.3s;  
+}
+.sw348reactSwitchButton{
+  height: 20px;
+  width: 20px;
+  filter: 'drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.15))';
```

Comment:
There is no shadow under mint design system. Please revisit the design and use case

Reply:
<img width="1102" alt="Screenshot 2023-09-30 at 11 54 26 AM" src="https://github.com/Groww/webster/assets/144772429/b17fe100-c323-484c-a480-bd3b0417df8a">
There is, under the circular button itself.